.github/release-drafter.yml

```diff
@@ -0,0 +1,51 @@
+# Configuration for Release Drafter: https://github.com/toolmantim/release-drafter
+name-template: LDAP-Auth Plugin v$NEXT_MAJOR_VERSION
```

Comment:
```suggestion
name-template: Version $NEXT_MAJOR_VERSION
```

I'm not sure about this release drafter thing, but let's give it a go.

Reply:
Short heads up: 
the first Draft will likely resolve a incorrect version from the earlier releases (likely will list as "Version 7") to prevent this from happening you could create a empty release for Version 9 so that the version resolver wont take Version 7 for latest.

Reply:
![image](https://user-images.githubusercontent.com/33120068/101076557-d71ee680-35a3-11eb-8615-69c3aa39b2a7.png)
This is what it looks like rn on my fork

Reply:
I would expect it to use the tags as a reference here, but I created releases for the v8 and v9 tags just in case.

Reply:
The version resolver uses the `latest`release to resolve the next version, so creating a empty V9 release is enough :wink: